orca-sql/src/main/kotlin/com/netflix/spinnaker/orca/sql/pipeline/persistence/SqlExecutionRepository.kt

```diff
@@ -440,6 +423,10 @@ class SqlExecutionRepository(
     buildTimeEndBoundary: Long,
     executionCriteria: ExecutionCriteria
   ): List<Execution> {
+    if (pipelineConfigIds.isEmpty()) {
```

Comment:
Seems reasonable, I'd also consider adding this to `retrieveAllPipelinesForPipelineConfigIdsBetweenBuildTimeBoundary()` since it's a public method on the interface (even though the current impl does delegate to this fn).

Reply:
👍